MH4U Dex/ItemMonsterSourcesTableViewController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  ItemMonsterSourcesTableViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/19/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "ItemMonsterSourcesTableViewController.h"
+
+#import "Item.h"
+#import "MonsterDrop.h"
+#import "Monster.h"
+
+#import "MonsterDetailsViewController.h"
+
+#import "ItemMonsterSourceTableViewCell.h"
+
+#import <CoreData/CoreData.h>
+
+@interface ItemMonsterSourcesTableViewController ()
+
+@property (nonatomic, strong) NSArray *sources;
+
+@end
+
+@implementation ItemMonsterSourcesTableViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    NSError *fetchError = nil;
+    NSFetchRequest *itemFetchRequest = [[NSFetchRequest alloc] initWithEntityName:@"Item"];
+    NSPredicate *itemPredicate = [NSPredicate predicateWithFormat:@"%K == %@", @"name", self.itemName];
+    [itemFetchRequest setPredicate:itemPredicate];
+    Item *item = (Item *)[self.managedObjectContext executeFetchRequest:itemFetchRequest error:&fetchError].firstObject;
+    
+    NSFetchRequest *fetchRequest = [[NSFetchRequest alloc] initWithEntityName:@"MonsterDrop"];
+    NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"id" ascending:YES];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropRelationships.item, item];
+    [fetchRequest setPredicate:predicate];
+    [fetchRequest setSortDescriptors:@[sortDescriptor]];
+    
+    self.sources = [self.managedObjectContext executeFetchRequest:fetchRequest error:&fetchError];
+    // If the fetch failed (most likely because the item cannot be found from any monsters) ...
+    if (!self.sources) {
+        // Set sources to empty.
+        self.sources = [NSArray arrayWithObjects:nil];
```

Comment:
Changed.  What about [NSArray array]?  (Or, for a mutable array, [NSMutableArray array]?)